server/controllers/wazuh-api.js

```diff
@@ -964,4 +965,10 @@ export class WazuhApiCtrl {
       return ErrorResponse(error.message || error, 3035, 500, reply);
     }
   }
+
+  // Get de list of available requests in the API
+  async getRequestList(req, reply) {
```

Comment:
Why is this function async? it must return an array of objects, so there are no async methods here. Also, I suggested refactoring util/api-request-list-json.js so take a look before editing this file.